frontend/src/components/pages/Logout.js

```diff
@@ -13,6 +14,9 @@ const mapDispatchToProps = dispatch => ({
 })
 
 function Logout({ logout }) {
+  const mainSocket = useContext(MainSocketContext)[0]
```

Comment:
Gibt der Context einen Array zurück? destructuring assignement würde besser aussehen, denke ich

Reply:
Habe ich in ein Object umgeschrieben und sieht auch besser aus! Danke! 👍